components/automate-ui/e2e/admin.e2e-spec.ts

```diff
@@ -1533,6 +1533,43 @@ describe('Admin pages', () => {
         });
       });
     });
+
+    describe('when the max of 6 projects is reached', () => {
```

Comment:
6 custom projects, or 6 total projects? If it's custom, can we include that in the code wherever possible?

Reply:
Or  . . . do we not display the chef-managed ones?

Reply:
Great question! Lemme look.

Reply:
Looks like they aren't included! Thanks for bringing that edge case up though :) 

https://github.com/chef/automate/blob/master/components/authz-service/server/v2/project.go#L202-L204

Reply:
oh, right. The chef-managed one is just internal